src/utils/otel/trace-context.ts

```diff
@@ -84,3 +84,32 @@ export function addTraceContextHttpHeaders(
    */
   fn.call(ctx, "traceparent", `00-${span.traceId}-${span.spanId}-01`);
 }
+
+export function addXRayTraceContextHttpHeaders(
```

Comment:
Suggestion: lets also rename `addTraceContextHttpHeaders` to something like `addW3CTraceContextHttpHeaders`